User_Service/database_config/config.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	data "github.com/E-Furqan/Food-Delivery-System/Data"
+	"github.com/joho/godotenv"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func Connection() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatalf("Error loading .env file: %v", err)
+	}
+
+	portstrr := os.Getenv("port")
+	port, err := strconv.Atoi(portstrr)
```

Comment:
we can create a environment variable struct and populate values there, then get those values here instead of doing processing here. DB connection should do one job, create connection

Reply:
have made the changes according db connection is on config while all other task in database  and have also implemented environment variable